packages/manager/apps/telecom/src/app/telecom/telephony/alias/portability/portabilities/attach/telecom-telephony-alias-portabilit-portabilities-attach.controller.js

```diff
@@ -0,0 +1,51 @@
+import endsWith from 'lodash/endsWith';
+
+angular.module('managerApp').controller('TelecomTelephonyServicePortabilityMandateAttachCtrl',
+  class TelecomTelephonyServicePortabilityMandateAttachCtrl {
+    constructor($q, $stateParams, $timeout, $translate, $uibModalInstance, OvhApiTelephony,
+      TucToast, data) {
+      this.$translate = $translate;
+      this.$uibModalInstance = $uibModalInstance;
+      this.TucToast = TucToast;
+      this.OvhApiTelephony = OvhApiTelephony;
+      this.$stateParams = $stateParams;
+      this.data = data;
+      this.$q = $q;
+      this.$timeout = $timeout;
+    }
+
+    $onInit() {
+      this.isLoading = false;
+      this.hasChecked = false;
+    }
+
+    attachMandate() {
+      this.isLoading = true;
+      return this.$q.all({
+        noop: this.$timeout(angular.noop, 1000),
```

Comment:
Why do you need to do this ?

Reply:
To be sure that the upload of the file is ok. I took example on upload document on telecom manager.

Reply:
It would be better to poll or add a bigger timeout when doing the call